web/viewer.css

```diff
@@ -291,7 +295,7 @@ html[dir='rtl'] #toolbarSidebar {
 
 #toolbarContainer, .findbar, .secondaryToolbar {
   position: relative;
-  height: 32px;
+  height: auto;
```

Comment:
It is necessary for `.findbar` at least to have a height of `auto` so that it can grow when we need to rows of things. I suppose keeping that separate from #toolbarContainer and .secondaryToolbar would be something we can change.